source/embedded-shell.txt

```diff
@@ -67,27 +67,20 @@ to run it.
         { $match: { "last_name": "Smith" } } // press Shift + Enter
       ] )                                    // Press Enter
 
-   .. seealso::
-
-      :pipeline:`$match`
-
 Learn More
 ~~~~~~~~~~
 
 The following links direct to the
-`MongoDB Shell Documentation
-<https://docs.mongodb.com/mongodb-shell/>`__, which contains more
-complete reference for the MongoDB Shell, including syntax and
+:ref:`mongosh documentation <mdb-shell-overview>`, which contains
+more a complete reference for :binary:`mongosh`, including syntax and
```

Comment:
Grammar issue. 

"which is a reference" or similar

Reply:
Done. 

I didn't want to change the existing content too much, it's often tempting but it can be a slippery slope.